tests/unit/models/360-core.js

```diff
@@ -96,17 +121,29 @@ describe("SerSol 360 Core Model >", function() {
     });
   });
 
-  describe("serSol360Core model getQueryVariable method >", function() {
-    it("should extract an issn from syndeticsImageUrl", function() {
-      var syndeticsImageUrl = titles[0].syndeticsImageUrl;
-      var issn = serSol360Core.getQueryVariable(syndeticsImageUrl, "issn");
-      expect(issn).toEqual("0028-4793");
+  describe("serSol360Core model getIssn method >", function() {
+    it("should extract an issn from the identifiers array when available", function() {
+      var title = titles[0];
+      expect(serSol360Core.getIssn(title)).toEqual("0028-4793");
+    });
+
+    it("should extract an eIssn from the identifiers array when an issn is unavailable", function() {
+      var title = titles[1];
+      expect(serSol360Core.getIssn(title)).toEqual("2163-307X");
+
+      var title = titles[2];
+      expect(serSol360Core.getIssn(title)).toEqual("1839-4901");
+    });
+
+    it("should return only an issn from the identifiers array when an issn is available and an eissn is not available", function() {
+      var title = titles[3];
+      expect(serSol360Core.getIssn(title)).toEqual("2960-1462");
     });
 
-    it("should return empty string when no issn is available on syndeticsImageUrl", function() {
-      var syndeticsImageUrl = scope.searchResultsCtrl.titleData.titles[2].syndeticsImageUrl;
-      var issn = serSol360Core.getQueryVariable(syndeticsImageUrl, "issn");
-      expect(issn).toEqual("");
+    it("should return an empty string if no issn or eissn is available", function() {
+      var titles = serSol360Core.getTitles(scope);
+      expect(serSol360Core.getIssn(titles[5])).toEqual("");
```

Comment:
Do you think we could improve the readability here a little by assigning `titles[5]` to a named variable that describes what it is?  Like
```
var journalWithEmptyStringForEissn = titles[5];
expect(serSol360Core.getIssn(journalWithEmptyStringForEissn)).toEqual("");
```

Then we can clearly see I the test what the difference is between that line, and the line with `titles[6]`, which could maybe be something like:
```
var journalWithNeitherEissnNorIssn = titles[6];
expect(serSol360Core.getIssn(journalWithNeitherEissnNorIssn)).toEqual("");
```

Reply:
Realized this probably indicated we needed separate test cases. Ended up splitting out the tests and adding on for the case where there is an empty value for an ISSN identifier.